pkg/apis/agones/v1/gameserver.go

```diff
@@ -277,15 +303,16 @@ func (gss *GameServerSpec) applyHealthDefaults() {
 	}
 }
 
-// applyStateDefaults applies state defaults
-func (gs *GameServer) applyStateDefaults() {
+// applyStatusDefaults applies Status defaults
+func (gs *GameServer) applyStatusDefaults() {
 	if gs.Status.State == "" {
 		gs.Status.State = GameServerStateCreating
-		// applyStateDefaults() should be called after applyPortDefaults()
+		// applyStatusDefaults() should be called after applyPortDefaults()
 		if gs.HasPortPolicy(Dynamic) || gs.HasPortPolicy(Passthrough) {
 			gs.Status.State = GameServerStatePortAllocation
 		}
 	}
+	gs.Status.Alpha.Players.Capacity = gs.Spec.Alpha.Players.InitialCapacity
```

Comment:
Should this be wrapped in an if statement? Since this is an alpha feature, presumably support for it will initially be off by default. But this code will always fill in a status field (with a value of 0 since spec won't be possible to set), right?

Reply:
You know, I completed forgot that feature flags where globals :man_facepalming: 

I will make this change :+1: